ur_description/urdf/ur.ros2_control.xacro

```diff
@@ -257,6 +261,70 @@
       </joint>
 
     </ros2_control>
+  </xacro:unless>
+
+  <xacro:if value="${gazebo_sim}">
+    <!-- Gazebo plugins -->
+    <gazebo reference="world">
+    </gazebo>
+    <gazebo>
+      <plugin filename="libgazebo_ros2_control.so" name="gazebo_ros2_control">
+        <parameters>${gazebo_controllers}</parameters>
+      </plugin>
+    </gazebo>
+
+    <!-- TODO: Implement initial positions support in Gazebo - check ros-controls/gazebo_ros2_control#100 -->
```

Comment:
This colon breaks my launch using the testing repos of galactic. I didn't dig deeper but removing it made launching possible. One older issue I found: https://github.com/ros2/launch_ros/issues/136